nmdc_runtime/site/ops.py

```diff
@@ -1241,3 +1246,58 @@ def ncbi_submission_xml_from_nmdc_study(
         all_instruments,
     )
     return ncbi_xml
+
+
+@op
+def nmdc_study_id_filename(nmdc_study_id: str) -> str:
```

Comment:
I think this needs a more specific name since it produces a very specific file name.

Reply:
Will address it!